### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/bastion ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +116,36 @@ If you are using Bastion open a PR so we can include it in our showcase.
 * [SkyNet](https://github.com/vertexclique/skynet) (a Discord bot which is resending deleted messages)
     * Skynet is running since 0.1.3 release of Bastion on the cloud and haven't killed yet.
 
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
+<a href="https://github.com/bastion-rs/bastion/graphs/contributors"><img src="https://opencollective.com/bastion/contributors.svg?width=890&button=false" /></a>
```
Comment (JohnTitor) [inline README.md]: I prefer markdown style to write HTML directly, something like:
```markdown
[![Code Contributors](https://opencollective.com/bastion/contributors.svg?width=890&button=false)](https://github.com/bastion-rs/bastion/graphs/contributors)
```

File: README.md [hunk 2]
```diff
@@ -116,6 +116,36 @@ If you are using Bastion open a PR so we can include it in our showcase.
...
+<a href="https://opencollective.com/bastion/organization/2/website"><img src="https://opencollective.com/bastion/organization/2/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/3/website"><img src="https://opencollective.com/bastion/organization/3/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/4/website"><img src="https://opencollective.com/bastion/organization/4/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/5/website"><img src="https://opencollective.com/bastion/organization/5/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/6/website"><img src="https://opencollective.com/bastion/organization/6/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/7/website"><img src="https://opencollective.com/bastion/organization/7/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/8/website"><img src="https://opencollective.com/bastion/organization/8/avatar.svg"></a>
+<a href="https://opencollective.com/bastion/organization/9/website"><img src="https://opencollective.com/bastion/organization/9/avatar.svg"></a>
```
Comment (JohnTitor) [inline README.md]: I think this `organizations` section should be the same as `individuals` and many link tags are unnecessary.

**Comment by JohnTitor** [comment]
Ping @monkeywithacupcake: could you make a change following my suggestions?

**Comment by vertexclique** [comment]
Closing due to inactivity. After addressing comments please feel free to open another PR.

**Pull request closed without merging**